api/models.py

```diff
@@ -21,6 +21,9 @@ class AssetSubCategory(models.Model):
     created_at = models.DateTimeField(auto_now_add=True, editable=False)
     last_modified = models.DateTimeField(auto_now_add=True, editable=False)
 
+    class Meta:
+        verbose_name_plural = 'Asset Categories'
```

Comment:
i merged this too soon.. please change this to `Asset Sub Categories`

Reply:
ooops,,,on it

Reply:
`Asset Subcategories` ie  😄

Reply:
@egichuri done

Reply:
pr?